File: Ass1/first.cs
using System;
class first
{
    static void Main(string[] args)
    {
        int i = 0, j = 5;
        char ch;
        ch = char.Parse(Console.ReadLine());
        Console.WriteLine("the ASCII code of '{0}' is {1}", ch, Convert.ToInt32(ch));
        Console.WriteLine("the value of i is {0}", i);
        i = ch;
        Console.WriteLine("the value of i is {0}", i);
    }
}

File: Ass1/first.java
import java.util.Scanner;
public class first{
    public static void main(String[] args) {
        int i = 0, j = 5;
        char ch;
        Scanner scan = new Scanner(System.in);
        ch = scan.next().toCharArray()[0];
        System.out.printf("the ASCII code of '%c' is %d\n", ch, (int)ch);
        System.out.printf("the value of i is %d\n", i);
        i = ch;
        System.out.printf("the value of i is %d\n", i);
    }
}